WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -77,14 +93,14 @@ public final class ProductsScreen: ScreenObject {
         return self
     }
 
-    public func tapAddProduct() throws -> Self {
```

Comment:
found another [function that does the exact same thing](https://github.com/woocommerce/woocommerce-ios/blob/e2a79762ff83658f213de07b6af71cd3ca8dc797/WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift#L43-L46) called `selectAddProduct` 😅 removing this as it's redundant